### ISSUE TITLE
README fails to render if repository field in package.json is an empty object

### ISSUE THREAD

**Issue by bcoe** [npm/marky-markdown#386]
```
 info storefile-transforms:/packages/n/ng-compiler/versions/version/0.0.7/contents/readme.md: marky-markdown: Parse markdown into HTML and add syntax highlighting
error storefile-transforms:/packages/e/ember-runkit/versions/version/0.3.0/contents/README.md: TypeError: repoUrl.match is not a function
  at module.exports (/mnt/deploys/storefile-transforms/packages/readme-md-to-readme-html/node_modules/github-url-to-object/dist/commonjs.js:16:27)
  at Function.module.exports (/mnt/deploys/storefile-transforms/packages/readme-md-to-readme-html/node_modules/marky-markdown/lib/plugin/github.js:52:14)
  at MarkdownIt.use (/mnt/deploys/storefile-transforms/packages/readme-md-to-readme-html/node_modules/markdown-it/lib/index.js:496:10)
  at Function.render.getParser (/mnt/deploys/storefile-transforms/packages/readme-md-to-readme-html/node_modules/marky-markdown/lib/render.js:83:6)
```

**Comment by revin** [comment]
This is a bug in github-url-to-object, but I think we can work around it temporarily.

**Comment by bcoe** [comment]
@revin I'm sure @zeke would land a patch fairly quickly, might be worth just helping out and patching it on `github-url-to-object`.

**Comment by zeke** [comment]
Would happily accept and release a fix for this. 👍

**Comment by revin** [comment]
@zeke [you got it](https://github.com/zeke/github-url-to-object/pull/29)

**Comment by zeke** [comment]
That fix has landed in github-url-to-object 4.0.0

**Comment by bcoe** [comment]
@zeke very much appreciate it 👍 ; I'm really excited about this work @revin has been doing to make things closer to github flavored markdown.
